bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v23/BitcoindV23RpcClientTest.scala

```diff
@@ -29,24 +27,11 @@ class BitcoindV23RpcClientTest extends BitcoindFixturesFundedCachedV23 {
       }
   }
 
-  it should "be able to decode a reedem script" in {
```

Comment:
removed this test as it doesn't work with descriptor wallets and it was just a copy from v22 tests